portality/view/admin.py

```diff
@@ -461,7 +461,9 @@ def application_quick_reject(application_id):
 def admin_site_search():
     #edit_formcontext = formcontext.ManEdBulkEdit()
     #edit_form = edit_formcontext.render_template()
-    edit_formulaic_context = JournalFormFactory.context("bulk_edit")
+    edit_formulaic_context = JournalFormFactory.context("bulk_edit", extra_param={
+        'cur_user': current_user,
```

Comment:
I would suggest keep `cur_user` remain unchanged.
because `disable_edit_note_except_cur_user` only accept current user that reading or generating that form.

that can avoid caller (flask, script or background job) wrongly input some other user like account of Journal creator  or account of Journal editor .

Reply:
ok, in which case let's be specific about the kind of user without using Flask semantics.  Let's call it something like `editing_user`.